online_meetings/report.py

```diff
@@ -151,7 +163,10 @@ def zoom_loop(url: str, headers: Dict[str, Union[str, int]], json_attribute_name
                 page_token = results.get("next_page_token")
                 params["next_page_token"] = page_token
             elif params.get("page_number"):
-                params["page_number"] += 1
+                if (isinstance(params["page_number"], int)):
+                    params["page_number"] += 1
+                else:
+                    raise TypeError("Could not increment page number as it not an int")
```

Comment:
Is this just some extra error handling, or were you responding to some particular case?

Reply:
With the mypy since it's `Union[str, int]` It needs a specific isinstance check or it flags it as an error. There's little to no chance of having a non-int case for this key but if it does come up I figure I'd throw the exception rather than continue on.